app/controllers/bookmarks_controller.rb

```diff
@@ -86,7 +86,7 @@ def bookmark_params
     permitted_attributes(@bookmark || Bookmark).tap do |obj|
       tags = Tag.find_or_create_tags tags: params.dig(:bookmark).fetch(:tags, [])
 
-      webpage = Webpage.upsert uri_string: params.dig(:bookmark, :uri_string)
+      webpage = Webpage.upsert uri: params.dig(:bookmark, :uri)
```

Comment:
you're doing gods work and your future ~critics~ self will thank you

Reply:
but its still labeled "url" on the frontend <_<